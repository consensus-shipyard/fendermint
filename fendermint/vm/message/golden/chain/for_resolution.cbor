a16d466f725265736f6c7574696f6ed82a58250001eec89b80b00f84a47619868141fce6d980c0ceaa89bae5fe7a12130a0acbcf00ea9663