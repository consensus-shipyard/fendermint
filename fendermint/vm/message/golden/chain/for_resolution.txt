ForResolution(Cid(bahxmrg4awahyjjdwdgdicqp443mybqgovke3vzp6pijbgcqkzphqb2uwmm))